contrib/python/pypodman/pypodman/lib/action_base.py

```diff
@@ -17,29 +17,21 @@ def subparser(cls, parent):
         Use set_defaults() to set attributes "class_" and "method". These will
         be invoked as class_(parsed_args).method()
         """
-        parent.add_argument(
+        parent.add_flag(
             '--all',
```

Comment:
As long as you're touching this, would you consider adding a `-a` alias for consistency with podman/docker?

Reply:
done